stylesheet.css

```diff
@@ -5,7 +5,6 @@
 html {
     font-size: 62.5%;
     font-family: helvetica;
```

Comment:
i would put font-family in the body but if it works then i guess it doesn't matter :)

Reply:
good call, moved it to the body.